shell.py

```diff
@@ -653,3 +654,33 @@ def setup_prometheus(region=default_region(), tag='prometheus', target_regions=u
 
     color_print('intalling prometheus')
     run_task('install-prometheus', regions=[region], parallel=False, tag=tag)
+
+
+def setup_smart_flooder(path_to_contract_repo, region=default_region(), tag='dev', pids=None):
+    shutil.copytree(path_to_contract_repo, './bin/contracts-cli', ignore=shutil.ignore_patterns('.git', 'node_modules', 'target'), dirs_exist_ok=True)
+
+    call('yarn'.split(), cwd='./bin/contracts-cli/deploy')
+    call('yarn redspot compile'.split(), cwd='./bin/contracts-cli/deploy')
+    call('cargo clean'.split(), cwd='./bin/contracts-cli/deploy')
+    call('zip -r ./repo.zip ./contracts-cli'.split(),  cwd='./bin')
+
+    color_print('setup ...')
+    run_task('setup-contract-repo', regions=[region], tag=tag, pids=pids)
+
+
+def start_smart_flooder(signer='//Alice', methods_to_call=None, n_of_calls=None, region=default_region(), tag='dev', pids=None):
+    script = '#!/usr/bin/env bash\n' \
```

Comment:
Perhaps we can use `"""<multi_line_string> """` here?

Reply:
done